packages/cashier/src/stores/payment-agent-transfer-store.ts

```diff
@@ -77,15 +96,15 @@ export default class PaymentAgentTransferStore {
     }
 
     @action.bound
-    setMinMaxPaymentAgentTransfer({ min_withdrawal, max_withdrawal }) {
+    setMinMaxPaymentAgentTransfer({ min_withdrawal, max_withdrawal }: TTransferLimit): void {
         this.transfer_limit = {
-            min: min_withdrawal,
-            max: max_withdrawal,
```

Comment:
- refactor transfer_limit variable (replace min, max to min_withdrawal, max_withdrawal) because of using one type for input parameters and `this.transfer_limit` variable itself like n the rest code

- add changes in account-transfer-form.jsx according new naming